internal/runbits/requirements/requirements.go

```diff
@@ -406,3 +411,19 @@ func initializeProject() (*project.Project, error) {
 
 	return project.FromPath(target)
 }
+
+func versionStringToRequirements(version string) ([]bpModel.VersionRequirement, error) {
+	constraints, err := model.VersionStringToConstraints(version)
+	if err != nil {
+		return nil, errs.Wrap(err, "Unable to process version string into constraints")
+	}
```

Comment:
Can we make this only act if we find that the value isn't just decimal numbers? Most of the time we shouldn't need to reach out to this API.

Reply:
This is done in a separate PR: https://github.com/ActiveState/cli/pull/2663/files#diff-d9612958a129acac4c7a4c03e241b6b912c286635b188a24975afe794fcc94f5R605-R618